allen-cell-animated/timelapse-colorizer
# feat: Updated export layout, video export now matches canvas dimensions

Problem
=======
Updates the export layout based on feedback from UX, and also prepares for #147, showing keys/legends in export.

This one should be a quick one, but is a couple different small changes rolled together.

*Estimated review size: small, 10 minutes*

Solution
========
- The current canvas dimensions are now used to determine the width and height of exported videos.
- Fixes a bug where the `vp09` codec would cause Export to fail if used.
- Changes layout on the `Export` modal to group all configuration settings.
- Allows SpinBox components to be resized.
- Increases contrast of hint text

## Type of change
Please delete options that are not relevant.

* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Open the preview link: https://allen-cell-animated.github.io/timelapse-colorizer/pr-preview/pr-443/
2. Click export and export a short video sequence.
3. Resize the window.
4. Click export again and export a short video sequence. The new video sequence should have a different resolution than the first one.

Screenshots (optional):
-----------------------
New export layout:
| Before | After |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/5a332a31-7610-4460-b6e6-36d0a1af2b38) | ![image](https://github.com/user-attachments/assets/294ed1a7-ff20-4b29-84ab-82fc10dfefa0) |

Large resolution:

https://github.com/user-attachments/assets/22083dfb-95a7-4ea4-8f61-80a678d39746

Small resolution:

https://github.com/user-attachments/assets/4749eaa4-2224-4529-8bdc-3e818646ea98
